packages/middleware-compression/src/compressionMiddleware.spec.ts

```diff
@@ -15,8 +15,8 @@ describe(compressionMiddleware.name, () => {
   const mockBody = "body";
   const mockConfig = {
     bodyLengthChecker: jest.fn().mockReturnValue(mockBody.length),
-    disableRequestCompression: false,
-    requestMinCompressionSizeBytes: 0,
+    disableRequestCompression: () => Promise.resolve(false),
```

Comment:
```suggestion
    disableRequestCompression: async () => false,
```

minor

Reply:
Fixed in [`04f0594` (#1132)](https://github.com/smithy-lang/smithy-typescript/pull/1132/commits/04f0594d40f10c22acd7a7e3a0c452462ba80775)